plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -586,6 +585,12 @@ protected JdbcOutputTableHandle createTable(ConnectorSession session, ConnectorT
         }
     }
 
+    protected void createTable(Connection connection, RemoteTableName remoteTableName, List<String> columns, ConnectorTableMetadata tableMetadata)
```

Comment:
Is it overridden in all subclasses, or in some?

Reply:
It's overridden in some (MariaDB and MySQL).

Reply:
We have now two extension points for subclasses: `createTable` and `createTableSql`.
It's generally better to have one extension point.
Would https://github.com/trinodb/trino/pull/14088#discussion_r968051558 allow us to revert this to single extension point, the `createTableSql` method?